Telegram.php

```diff
@@ -100,14 +100,11 @@ public function __construct($bot_token, $log_errors = true, array $proxy = [])
      * \param $post boolean tells if $content needs to be sends
      * \return the JSON Telegram's reply.
      */
-    public function endpoint($api, array $content, $post = true)
+    public function endpoint($api, array $content = [], $post = true)
     {
-        $url = 'https://api.telegram.org/bot'.$this->bot_token.'/'.$api;
-        if ($post) {
-            $reply = $this->sendAPIRequest($url, $content);
-        } else {
-            $reply = $this->sendAPIRequest($url, [], false);
-        }
+        $url = 'https://api.telegram.org/bot' . $this->bot_token . '/' . $api;
+
+        $reply = $this->sendAPIRequest($url, $content, $post);
```

Comment:
Isn't it is unnecessary to create a `$reply` variable after the removal of `if` statement.